cypress/integration/breadcrumbs.spec.js

```diff
@@ -0,0 +1,95 @@
+const constants = {
+    rootUrl: 'https://www.bcpl.info/youth/index.html',
+    blogUrl: 'https://www.bcpl.info/blog',
+    selectors: {
+        blogEntry: '.card--blog .card-heading',
+        breadcrumbs: '.breadcrumbs-wrapper .breadcrumb',
+        collapsibleBreadCrumbContainer: '.hidden-breadcrumb-container',
+        hiddenBreadCrumbList: '.hidden-breadcrumb-list'
+    }
+};
+
+describe('Breadcrumbs', () => {
+    beforeEach(() => {
+        cy.get(constants.selectors.breadcrumbs).as('breadCrumbs');
+    });
+
+    describe('default', () => {
+        before(() => {
+            cy.visit(constants.rootUrl);
+        });
+
+        it('should show two breadcrumbs', () => {
+            cy
+                .get('@breadCrumbs')
+                .its('length')
+                .should('eq', 2);
+        });
+
+        it('should the first breadcrumb as a link "Home"', () => {
+            cy
+                .get('@breadCrumbs')
+                .first()
+                .contains('Home');
+        });
+
+        it('should the first breadcrumb as a link "Youth"', () => {
```

Comment:
Same as above. I think this should be _last_ instead of _first._

Reply:
@jsmith176 updated